src/pages/LocalAuthorityDashboard/LocalAuthorityDashboard.module.scss

```diff
@@ -62,3 +83,25 @@
     margin: 40px 15px 40px 0;
   }
 }
+
+@media only screen and (max-width: g.$screen-large) {
```

Comment:
Not completely necessary, but I do have breakpoints mixins in the globals. Just saves a bit of code - have a search and see how I've implemented `@include g.$sm-breakpoint { ...` for eg